SDK-270 [Node.js SDK] Update README introduction section - more readme updates (#174)
incountry/sdk-nodejs: README.md

#### Encryption key/secret

> The `getSecrets` storage config option allows you to pass function which will be used to fetch an encryption key(s) or secret. This function should return either a string representing your secret or an object (we call it `SecretsData`) or a `Promise` which is resolved to that string or object:

Reviewer: ```suggestion
The `getSecrets` storage config option allows you to pass function which will be used to fetch an encryption key(s) or secret. This function should return either a string representing your secret or an object (so-called `SecretsData`) or a `Promise` which is resolved to such string or object:
```
